app/src/main/java/org/oppia/android/app/profile/AddProfileViewModel.kt

```diff
@@ -22,7 +22,7 @@ class AddProfileViewModel @Inject constructor(
   val createPin = ObservableField(false)
   val isButtonActive = ObservableField(false)
   val showInfoAlertPopup = ObservableField<Boolean>(false)
-  val requiredTextWithStar: String = "*" +
+  val requiredField: String =
     resourceHandler.getStringInLocale(R.string.add_profile_required)
```

Comment:
Nit: can this fit on one line now?

Reply:
Yes, however, this line has now been removed as per the comment below.